internal/guestrequest/types.go

```diff
@@ -49,6 +49,10 @@ type LCOWMappedVPMemDevice struct {
 	MountPath    string `json:"MountPath,omitempty"`
 }
 
+type LCOWMappedVPciDevice struct {
+	VmbusUuid string `json:"VmbusUuid,omitempty"`
```

Comment:
Typically in Go you write acronyms/initialisms in either all upper or all lower case. I would rewrite this as `VMBusUUID`.

Reply:
https://github.com/golang/go/wiki/CodeReviewComments#initialisms